reference/payment_processing.yml

```diff
@@ -911,6 +926,18 @@ components:
                         last_4: '4242'
                         token: 050a1e5c982e5905288ec5ec33f292772762033a0704f46fccb16bf1940b51ef
                         is_default: true
+                  - id: bigcommerce.gift_certificate
+                    name: Gift Certificate
+                    test_mode: false
+                    type: gift_certificate
+                    supported_instruments: []
+                    stored_instruments: []
+                  - id: bigcommerce.store_credit
+                    name: Store Credit
+                    test_mode: false
+                    type: store_credit
+                    supported_instruments: []
+                    stored_instruments: []
```

Comment:
Can we add these before `stripe`, as they are sorted alphabetically in the actual API response.